src/gt4py/next/iterator/embedded.py

```diff
@@ -744,48 +733,51 @@ def _make_tuple(
     named_indices: NamedFieldIndices,
     *,
     column_axis: Optional[Tag] = None,
-) -> Column | npt.DTypeLike | tuple[tuple | Column | npt.DTypeLike, ...]:
-    column_range = column_range_cvar.get()
-    if isinstance(field_or_tuple, tuple):
-        if column_axis is not None:
-            assert column_range
-            # construct a Column of tuples
-            first = tuple(
-                _make_tuple(f, _single_vertical_idx(named_indices, column_axis, column_range.start))
-                for f in field_or_tuple
-            )
-            col = Column(
-                column_range.start, np.zeros(len(column_range), dtype=_column_dtype(first))
-            )
-            col[0] = first
-            for i in column_range[1:]:
-                col[i] = tuple(
-                    _make_tuple(f, _single_vertical_idx(named_indices, column_axis, i))
-                    for f in field_or_tuple
-                )
-            return col
-        else:
+) -> Column | npt.DTypeLike | tuple[tuple | Column | npt.DTypeLike, ...] | Undefined:
```

Comment:
Just a question: shouldn't  the overloads accepting LocatedField return an union including `Undefined` ?

Reply:
yes

Reply:
adding it introduced a type error that I didn't want to sort out now. added a TODO...

Reply:
found the problem..